src/extensions/extension-loader.ts

```diff
@@ -71,21 +93,27 @@ export class ExtensionLoader {
   }
 
   protected autoInitExtensions(register: (ext: LensExtension) => Function[]) {
-    return reaction(() => this.extensions.toJS(), (installedExtensions) => {
-      for (const [id, ext] of installedExtensions) {
-        let instance = this.instances.get(ext.manifestPath)
-        if (!instance) {
-          const extensionModule = this.requireExtension(ext)
-          if (!extensionModule) {
-            continue
-          }
+    return reaction(() => this.toJSON(), (installedExtensions) => {
+      for (const [extId, ext] of installedExtensions) {
+        let instance = this.instances.get(extId);
+        if (ext.enabled && !instance) {
           try {
+            const extensionModule = this.requireExtension(ext)
+            if (!extensionModule) continue;
             const LensExtensionClass: LensExtensionConstructor = extensionModule.default;
```

Comment:
`requireExtension`'s return value should be `{ default: LensExtensionConstructor }` to avoid this.